core/include/moveit/task_constructor/stages/action_base.h

```diff
@@ -0,0 +1,124 @@
+/*********************************************************************
+ * BSD 3-Clause License
+ *
+ * Copyright (c) 2020 PickNik LLC.
```

Comment:
```suggestion
 * Copyright (c) 2020 PickNik Inc.
```

Reply:
The LLC was updated to INC in both header files